### PR TITLE
Payment creation - Bugfixes

### LINKED ISSUE #143: Total of expense sheet is never calculated
It does not update when you alter the expense sheet

### PR DESCRIPTION
**Pull request by lukasbischof**
~does not close anymore #145~
~This pr implements payment creation~
This pr fixes bugs needed to start with payment creation

closes #143

### CODE REVIEW

File: config/locales/de-CH.yml [hunk 1]
```diff
@@ -140,7 +145,7 @@ de-CH:
         invalid_date: ungültiges Datum
         not_a_friday: kein Freitag
         not_a_monday: kein Montag
-        not_an_unsigned_integer: sind keine positive Zahlen
+        not_a_positive_currency_amount: sind keine positive Zahlen
```
Comment (TheFehr) [inline config/locales/de-CH.yml:148]: *sind keine positiven Zahlen or is my german broken?

File: app/models/service_specification.rb [hunk 2]
```diff
@@ -78,11 +78,11 @@ def validate_json_format(attribute)
   def validate_numericality_of_json(attribute)
     return if self[attribute].blank?
 
-    errors.add(attribute, :not_an_unsigned_integer) unless values_numeric?(self[attribute].values)
+    errors.add(attribute, :not_a_positive_currency_amount) unless values_numeric?(self[attribute].values)
```
Comment (TheFehr) [inline app/models/service_specification.rb:81]: But wouldn't this add it for -5 as well? Or is numeric always positiv?

Comment (lukasbischof) [reply app/models/service_specification.rb:81]: Just bad method name

File: config/locales/de-CH.yml (hunk 1 above)
Comment (lukasbischof) [reply config/locales/de-CH.yml:148]: Probably yes

**Review by TheFehr — APPROVED** [review]

**Pull request merged**